Automattic/jetpack
# Hide backup submenu if backup plugin is active

Jetpack adds a submenu to wp-admin when the site has Backup and or Scan active.

![Captura de Tela 2021-09-22 às 16 50 57](https://user-images.githubusercontent.com/971483/135360278-34d2d008-fb8d-4f2e-b6f1-487a1679845c.png)

This menu takes the user to Jetpack.com to manage their Backup and Scan configuration, and its label can be either `Backup`, `Scan` or `Backup & Scan` depending on which product is active.

This PR will hide `Backup` from this menu entry if the Backup plugin is also active, leaving only Scan.

#### Changes proposed in this Pull Request:

* Hide Backup submenu if the backup plugin is active

#### Jetpack product discussion


p1632340119030400-slack-C025MP1Q3HT

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Note: after buying or canceling a product, it might take a little while for the changes to take effect on the site

* On a new site, buy the Backup product
* Check that the menu item "Backup" is added to the Jetpack submenu
* Activate the Backup plugin
* Verify that the Backup menu item is no longer there
* Purchase the Scan product
* Verify that the Scan menu item is added to the menu
* Deactivate the Backup plugin
* Verify that now the menu item reads "Backup & Scan"
* Cancel the Backup subscription
* Verify that the menu item reads only "Scan"
* Activate the Backup plugin
* Verify that the "Scan" menu item is still there
